sources/packages/api/src/route-controllers/student-scholastic-standings/models/student-scholastic-standings.dto.ts

```diff
@@ -0,0 +1,26 @@
+import { IntersectionType } from "@nestjs/swagger";
+import { IsNotEmptyObject } from "class-validator";
+import { ActiveApplicationDataAPIOutDTO } from "../../../route-controllers/institution-locations/models/application.dto";
+
+export class ScholasticStandingDataAPIInDTO {
```

Comment:
As we talked about, let's convert it to an interface with necessary API properties only to allow the form to be changed dynamically.